test/unittest/test_variables.go

```diff
@@ -245,6 +245,22 @@ var DbVariableWithInvalidDBMasterPassword = map[string]interface{}{
 	"db_master_password":  "123@",
 }
 
+// Ingress
+
```

Comment:
Remove carriage return

Reply:
This is following the same pattern as other sections.